packages/internal/bridge/bridge-sample-app/src/withdraw.ts

```diff
@@ -27,7 +38,141 @@ async function withdraw() {
 
   const tokenBridge = new TokenBridge(bridgeConfig);
 
-  console.log('Deposit smoke tests TBC');
+  const rootBridge: ethers.Contract = getContract("RootERC20BridgeFlowRate", params.rootBridgeAddress, params.rootProvider);
+  const childBridge: ethers.Contract = getContract("ChildERC20Bridge", params.childBridgeAddress, params.childProvider);
+
+  let rootBridgeChildAddress = await rootBridge.rootTokenToChildToken(params.sepoliaToken);
+  let childBridgeChildAddress = await childBridge.rootTokenToChildToken(params.sepoliaToken);
+
+  if (rootBridgeChildAddress === ethers.constants.AddressZero
+    || childBridgeChildAddress === ethers.constants.AddressZero) {
+    console.log('token not mapped, please map token before withdrawing');
+    return;
+  }
+
+  if (childBridgeChildAddress === ethers.constants.AddressZero) {
+    console.log('token mappinng incomplete, please wait for token to map to childBridge before withdrawing');
```

Comment:
as a sanity check it might be good to check that the address of the child token returned from the root bridge matches that returned by the child bridge. Our last deployment issue involved a mismatch in these two, which shouldn't happen